src/screens/Drawer/index.js

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react';
+import { View, Text,Image } from 'react-native';
+import { NavigationContainer } from '@react-navigation/native';
+import { createDrawerNavigator, DrawerContentScrollView, DrawerItemList } from '@react-navigation/drawer';
+import { createKeyboardAwareNavigator } from 'react-navigation';
+import FontAwesome from 'react-native-vector-icons/FontAwesome';
+
+
+
+
+function Home() {
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+      <Text>Home Screen</Text>
+    </View>
+  );
+}
+function Profile() {
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+      <Text>Profile Screen</Text>
+    </View>
+  );
+}
+function MyListings() {
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+      <Text>My Listings Screen</Text>
+    </View>
+  );
+}
+function Forums() {
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+      <Text>Forums Screen</Text>
+    </View>
+  );
+}
+function Settings() {
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+      <Text>Settings Screen</Text>
+    </View>
+  );
+}
+
+function Help() {
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+      <Text>Help Screen</Text>
+    </View>
+  );
+}
+function Logout() {
+  return (
+    <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+      <Text>Logout Screen</Text>
+    </View>
+  );
+}
+
+
+const NavigationDrawer = createDrawerNavigator();
+
+const CustomDrawer = (props) => {
+return  <DrawerContentScrollView {...props}>
+  <View>
+    <View>
+    </View>
+    <Image source={require('../Drawer/DraweImage/image.png')} style={{width:280 , height:200 , borderBottomWidth:20}}/>
```

Comment:
image should be move to image folder in assets. please follow the folder structure